x/nexus/keeper/transfer.go

```diff
@@ -96,40 +98,50 @@ func (k Keeper) ComputeTransferFee(ctx sdk.Context, sourceChain exported.Chain,
 }
 
 // EnqueueTransfer enqueues an asset transfer to the given recipient address
-func (k Keeper) EnqueueTransfer(ctx sdk.Context, senderChain exported.Chain, recipient exported.CrossChainAddress, asset sdk.Coin) (exported.TransferID, error) {
+func (k Keeper) EnqueueTransfer(ctx sdk.Context, senderChain exported.Chain, receipient exported.CrossChainAddress, asset sdk.Coin) (exported.TransferID, error) {
```

Comment:
```suggestion
func (k Keeper) EnqueueTransfer(ctx sdk.Context, senderChain exported.Chain, recipient exported.CrossChainAddress, asset sdk.Coin) (exported.TransferID, error) {
```
why mess up with the spelling lol?

Reply:
lmao, i wish i knew. maybe i was thinking of receiver or something